test/unit/main/interactive_shell.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+#####################
+#! \file interactive_shell.py
+## \verbatim
+## Top contributors (to current version):
+##   Andrew V. Jones
+## This file is part of the CVC4 project.
+## Copyright (c) 2020 by the authors listed in the file AUTHORS
+## in the top-level source directory) and their institutional affiliations.
+## All rights reserved.  See the file COPYING in the top-level source
+## directory for licensing information.\endverbatim
+##
+## \brief A simpel test file to interact with CVC4 with line editing
+#####################
+
+import pexpect
```

Comment:
Could we make this import conditional? If the user has the dependency, we run the test, otherwise we skip it.

Reply:
Done!